src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/instance/sendPulse.st

```diff
@@ -0,0 +1,6 @@
+controlling
+sendPulse
+	triggerPin beDigitalOutput. 
+	triggerPin value: 0; value: 1.
+	1 milliSeconds wait.
+	triggerPin value: 0
```

Comment:
Do you know the pulse logic is always like that?
We can move this method to the pin itself. So it would be
```Smalltalk
triggerPin sendPulse
```
(but beDigitalOutput is part of connection logic)